File: java/src/main/java/io/cucumber/gherkin/GherkinLanguageConstants.java
package io.cucumber.gherkin;

interface GherkinLanguageConstants {
    char TAG_PREFIX_CHAR = '@';
    char COMMENT_PREFIX_CHAR = '#';
    char TITLE_KEYWORD_SEPARATOR = ':';
    /**
     * Size of TITLE_KEYWORD_SEPARATOR = 1 char.
     */
    int TITLE_KEYWORD_SEPARATOR_LENGTH = 1;
    char TABLE_CELL_SEPARATOR = '|';
    String DOCSTRING_SEPARATOR = "\"\"\"";
    String DOCSTRING_ALTERNATIVE_SEPARATOR = "```";
    String DEFAULT_LANGUAGE = "en";
}

File: php/src/GherkinLanguageConstants.php
<?php

namespace Cucumber\Gherkin;

interface GherkinLanguageConstants
{
    public const string TAG_PREFIX = "@";
    public const string COMMENT_PREFIX = "#";
    public const string TITLE_KEYWORD_SEPARATOR = ":";
    public const string TABLE_CELL_SEPARATOR = "|";
    public const string DOCSTRING_SEPARATOR = "\"\"\"";
    public const string DOCSTRING_ALTERNATIVE_SEPARATOR = "```";
}
